readme.md

```diff
@@ -1,58 +1,119 @@
-# Pawn
+# Pawn Community Compiler
 
-[![Build Status][build_status]][build] [![Build Status - Windows][build_status_win]][build_win]
+[![Build Status][build_status]][build]
+[![Build Status - Windows][build_status_win]][build_win]
+
+## What
+
+This is a modified copy of the Pawn compiler version 3.2.3664 by Compuphase that
+fixes some bugs and adds a few features.
+
+This project was originally maintained and managed by Zeex who left the project
+due to lack of time on 2017-12-31. Thank you for all your hard work on keeping
+this project alive Zeex!
+
+The project is now maintained by the
+[Compiler Team](https://github.com/orgs/pawn-lang/teams/compiler) at the
+pawn-lang GitHub org.
```

Comment:
These frequent unnecessary line breaks are very ugly when viewed in an editor (even though markdown treats them to be part of a single line).

Reply:
This is markdown lint rules, wrapping at column 80. My editor does it automatically.

Reply:
I'll move all the existing links to references at the end of the document to clear this up.